tests/framework/pvc.go

```diff
@@ -240,6 +240,35 @@ func (f *Framework) VerifySparse(namespace *k8sv1.Namespace, pvc *k8sv1.Persiste
 	return info.VirtualSize >= info.ActualSize, nil
 }
 
+// VerifyFSOverhead checks whether virtual size is smaller than actual size. That means FS Overhead has been accounted for
+func (f *Framework) VerifyFSOverhead(namespace *k8sv1.Namespace, pvc *k8sv1.PersistentVolumeClaim) (bool, error) {
+	var executorPod *k8sv1.Pod
+	var err error
+
+	executorPod, err = f.CreateVerifierPodWithPVC(namespace.Name, pvc)
+	if !apierrs.IsAlreadyExists(err) {
+		gomega.Expect(err).ToNot(gomega.HaveOccurred())
+	}
+	err = utils.WaitTimeoutForPodReady(f.K8sClient, executorPod.Name, namespace.Name, utils.PodWaitForTime)
+	gomega.Expect(err).ToNot(gomega.HaveOccurred())
+
+	cmd := fmt.Sprintf("qemu-img info %s/disk.img --output=json", utils.DefaultPvcMountPath)
+	output, stderr, err := f.ExecShellInPod(executorPod.Name, namespace.Name, cmd)
+
+	if err != nil {
+		fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: stderr: [%s]\n", stderr)
+		return false, err
+	}
+	fmt.Fprintf(ginkgo.GinkgoWriter, "INFO: qemu-img info output %s\n", output)
+	var info image.ImgInfo
+	err = json.Unmarshal([]byte(output), &info)
+	if err != nil {
```

Comment:
do we return something meaningful in case of err != nil?
why not return something like false, err ?

Reply:
Good point.